docs/source/markdown/podman-wait.1.md.in

```diff
@@ -14,6 +14,11 @@ name or ID.  In the case of multiple containers, Podman waits on each consecutiv
 After all specified containers are stopped, the containers' return codes are printed
 separated by newline in the same order as they were given to the command.
 
+Please note that there is an inherent race condition between waiting for a
+container and its restart policy, in particular containers created with the
+`always` restart policy which is used by default in `podman kube play`.  On
+return of `podman wait`, the container may be running or have exited again.
```

Comment:
I like the gist of it, but the reality is much more nuanced: the `on-failure` policy is just as dangerous. I genuinely believe that `podman wait` is meaningless, undefined, and misleading when run against any container with a restart-policy other than `no`. And I think we should be clear about that, perhaps even have `podman wait` check the restart policy and abort immediately. Or warn loudly. But if we can't, because docker, then we should be clear about it in the man page.
>NOTE: there is an inherent race condition when waiting for containers with a restart policy of `always` or `on-failure`, such as those created by `podman kube play`. Such containers may be repeatedly exiting and restarting, possibly with different exit codes, but `podman wait` can only display and detect one.

Reply:
I like your suggestion: applied as is :heavy_check_mark: